__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  it("status 200, should return an array of topic objects, each of which should have the following properties:slug ,description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        topics.forEach((topic) => {
```

Comment:
This forEach is a good way of asserting each object has the correct properties 👍  but to avoid a false positive we need to make sure that the array we are attempting to use the forEach on has a length (ie is NOT an empty array). Please add this assertion before the forEach assertion (can be added to this test).